connect/runtime/src/test/java/org/apache/kafka/connect/integration/ExactlyOnceSourceIntegrationTest.java

```diff
@@ -81,6 +81,7 @@
 import static org.apache.kafka.connect.integration.MonitorableSourceConnector.CUSTOM_EXACTLY_ONCE_SUPPORT_CONFIG;
 import static org.apache.kafka.connect.integration.MonitorableSourceConnector.CUSTOM_TRANSACTION_BOUNDARIES_CONFIG;
 import static org.apache.kafka.connect.integration.MonitorableSourceConnector.MESSAGES_PER_POLL_CONFIG;
+import static org.apache.kafka.connect.integration.MonitorableSourceConnector.THROUGHPUT_CONFIG;
```

Comment:
this could be THROUGHPUT_MSGS_PER_SEC_CONFIG

Reply:
I think that would make sense if the underlying configuration was `throughput.msgs.per.sec` but it is currently `throughput`. I preferred to keep the existing name instead of renaming + aliasing the configuration. just to keep this PR small.

Do you think renaming the configuration is important here?

Reply:
I had to go and look what unit `throughput` was in and `ThroughputThrottler` says `Can be messages/sec or bytes/sec` . In the case of this test it is messages, so for me teh longer name I suggested helps readability.